lib/Runtime/ByteCode/ByteCodeEmitter.cpp

```diff
@@ -6893,7 +6987,14 @@ void EmitMethodFld(ParseNode *pnode, Js::RegSlot callObjLocation, Js::PropertyId
     bool isScoped = (byteCodeGenerator->GetFlags() & fscrEval) != 0 ||
         (isRoot && callObjLocation != ByteCodeGenerator::RootObjectRegister);
 
-    EmitMethodFld(isRoot, isScoped, pnode->location, callObjLocation, propertyId, byteCodeGenerator, funcInfo, registerCacheIdForCall);
```

Comment:
This happens when we're trying to emit a call target when the symbol is an imported binding. We try to emit it as some kind of LdRootMethodFld but should be getting the method from the module storage instead. I can make the logic a little cleaner here, but not sure how to remove it completely.